config/terraform/application/application.tf

```diff
@@ -33,8 +33,86 @@ module "application_configuration" {
   )
 }
 
+# Run database migrations
+# https://guides.rubyonrails.org/active_record_migrations.html#preparing-the-database
+# Terraform waits for this to complete before starting web_application and worker_application
+resource "kubernetes_job" "migrations" {
+  metadata {
+    name      = "${var.service_name}-${var.environment}-migrations"
+    namespace = var.namespace
+  }
+
+  spec {
+    template {
+      metadata {
+        labels = { app = "${var.service_name}-${var.environment}-migrations" }
+        annotations = {
+          "logit.io/send" = "true"
```

Comment:
can you add "fluentbit.io/exclude" = "true"
as it stops us writing azure container logs alongside logit logs.